src/mlpack/tests/ann_dist_test.cpp

```diff
@@ -0,0 +1,157 @@
+/**
+ * @file ann_dist_test.cpp
+ * @author Atharva Khandait
+ *
+ * Tests the ann distributions.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/core.hpp>
+
+#include <mlpack/methods/ann/dists/normal_distribution.hpp>
+#include <mlpack/methods/ann/init_rules/random_init.hpp>
+
+#include <boost/test/unit_test.hpp>
+#include "test_tools.hpp"
+
+using namespace mlpack;
+using namespace mlpack::ann;
+
+BOOST_AUTO_TEST_SUITE(ANNDistTest);
+
+/**
+ * Simple normal distribution module test.
+ */
+BOOST_AUTO_TEST_CASE(SimpleNormalDistributionTest)
+{
+  static const constexpr double log2pi = 1.83787706640934533908193770912475883;
+  arma::mat param, target, gradient;
+  double output;
+  param.ones(10, 1);
```

Comment:
Just a note you can also write ``arma::mat param = arma::ones(10, 1)``.

Reply:
Just a doubt, is this faster than `param.ones(10, 1);`?

Reply:
It probably saves the copy constructor, but that's neglectable here.